### ISSUE TITLE
Vertical tab panel is too slow to auto expand

### ISSUE THREAD

**Issue by arkydon** [brave/brave-browser#42238]
### Description

It takes too long to autoexpand the vertical tab sidebar.

I was using Zen and Microsoft Edge browsers before this and they are much faster at opening the vertical tabs sidebar.

### Steps to reproduce

1. Enable Vertical Tabs
2. Hide the vertical tabs panel
3. Drag mouse to vertical tab sidebar to expand it.

### Actual result

Takes almost a second to open. 

### Expected result

Vertical tab panel should autoexpand instantly

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Brave: 1.74.3 Chromium: 131.0.6778.39 (Official Build) beta (64-bit)

Rev: 0ce9628041e549979e5c522f7ec10ee1f5470182

OS: Linux

### Channel information

- [ ] release (stable)
- [x] beta
- [ ] nightly

### Reproducibility

- [x] with Brave Shields disabled
- [x] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by B20bob** [comment]
I agree. I was searching around to see if there was a hidden setting to fix this when I cam across this post. Brave devs, if you could please fix this we would really appreciate it. This is the only thing preventing me from switching from Arc.

**Comment by okulvitra** [comment]
Backing this issue as well.

I'm a big fan of vertical tabs, but coming from Arc/Zen Browser, Brave vertical tabs opening feels sluggish at best or buggy. Having a responsive tab sidebar would be really the best.

**Comment by bnwa** [comment]
On my Windows box the auto expand is pretty snappy, but on macOS its not

**Comment by shakeelansari63** [comment]
I agree, I am using brave on linux. And it seems quite slow to expand in hover. I have used Zen browser and their vertical tab bar expansion is quite snappy. 

It really looks like there is some delayed start to expand on hover animation.

**Comment by NyxErinys** [comment]
I noticed this too, was thinking about switching back to a chromium based browser from zen due to some issues firefox as a whole, this issue was the first thing I noticed on trying the vertical tabs. 
I'm also on linux but my partner tried it on windows and was having the same issue.

**Comment by AshkanArabim** [comment]
This may be a duplicate of this issue: https://github.com/brave/brave-browser/issues/41184

**Comment by sangwoo108** [comment]
6Hello, everyone. Let me close this as duplicated.
